CustomDimensions.php

```diff
@@ -280,10 +281,13 @@ public function addConversionInformation(&$conversion, $visitInformation, Tracke
 
         foreach ($dimensions as $dimension) {
             $index = (int) $dimension['index'];
-            if ($dimension['scope'] === self::SCOPE_VISIT && in_array($index, $conversionIndexes)) {
+            if (in_array($index, $conversionIndexes)) {
                 $field = LogTable::buildCustomDimensionColumnName($dimension);
 
-                if (array_key_exists($field, $visitInformation)) {
+                if ($dimension['scope'] === self::SCOPE_ACTION && array_key_exists($field, $actionDimensions)) {
+                    $conversion[$field] = $actionDimensions[$field];
+                }
+                else if ($dimension['scope'] === self::SCOPE_VISIT  && array_key_exists($field, $visitInformation)) {
```

Comment:
I would not mind doing some more work on this issue, but lately I have been very busy so I can't say when I'll have the time for this. Also, I could use some pointers on what to change in the archiver. I already tried a few things but didn't get the expected result (probably just missing something).